packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts

```diff
@@ -87,6 +88,20 @@ export class LayeredRuntimeResolverCreator {
         }
       }
 
+      // If delta marker is encountered, add resolver for `delta` query
+      if (parseMarker('delta', resolverElement.graphqlType.description)) {
+        const diffQuery = `get${resolverElement.graphqlType.name}Delta`
```

Comment:
This is likely going to be used elsewhere, so we should centralise the naming in `mappingHelpers`, similar to [getSubscript](https://github.com/aerogear/graphback/blob/cf09a8d914a274c2344d7a8de2e66fc5e597b37e/packages/graphback-core/src/crud/mappingHelpers.ts#L77)

Reply:
`get<Model>Delta` implies this should only return a single item, since `get<Model>` is used to fetch a single record by its ID.

This should be named something different, like `sync<Model>s`.

Reply:
This is also done: https://github.com/aerogear/graphback/blob/bf46cbf020736b3edfe55b2632a369a736f40270/packages/graphback-core/src/crud/mappingHelpers.ts#L178